src/main.rs

```diff
@@ -94,13 +94,27 @@ fn main() {
         if let Some(mut args) = c.java_args {
             java_args.append(&mut args);
         }
+
+        if let Some(path) = c.wurst_path {
+            if File::open(&path).is_err() {
```

Comment:
Isn't `wurst_path` supposed to be a directory? In which case I think opening it as a file should always err, right?

Can you do the `format!` to construct the path before doing this open test?